src/univalue/include/univalue.h

```diff
@@ -304,4 +306,9 @@ extern const UniValue NullUniValue;
 
 const UniValue& find_value(const UniValue& obj, const std::string& name, const bool throw_if_not_present = false);
 
+class KeyNotPresentError: public std::runtime_error {
+    public:
+        KeyNotPresentError(): std::runtime_error("KeyNotPresentError") {}
```

Comment:
These exception constructors are usually made `explicit` to prevent implicit conversion. You can see [uint_error](https://github.com/bitcoin/bitcoin/blob/master/src/arith_uint256.h#L18-L21) as an example.